### PR TITLE
Table data update

### PR DESCRIPTION
**Pull request by pichar376**

### CODE REVIEW

File: tatuds [hunk 1]
```diff
@@ -0,0 +1,35 @@
+[33mcommit 6fa80f6087e6452ed2f3805c9e86a9b6138cea6e[m[33m ([m[1;36mHEAD -> [m[1;32mfooter[m[33m, [m[1;31morigin/footer[m[33m)[m
```
Comment (r3ivaj) [inline tatuds]: I think you accidentally committed this file

File: src/components/ButtonTable/ButtonTable.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { Typography } from "@mui/material";
+import { buttonDetailStyle } from "../../buttonDelivery.styles";
+import Button from "../Button/Button";
+
+const ButtonTable = ({ Icon, children }) => {
```
Comment (r3ivaj) [inline src/components/ButtonTable/ButtonTable.jsx]: A better name for this should be SecondaryButton. What if we want to use this button in another place that is not the table?

Also it shouldn't contain the `endIcon`, this `endIcon` should be passed as prop

Comment (r3ivaj) [reply src/components/ButtonTable/ButtonTable.jsx]: Example:
Screenshot 2022-12-06 at 10 10 22 p m

File: src/components/MenuDelivery/MenuDelivery.jsx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+        backgroundColor: alpha(
+          theme.palette.primary.main,
+          theme.palette.action.selectedOpacity
+        ),
+      },
+    },
+  },
+}));
+
+const MenuDelivery = () => {
```
Comment (r3ivaj) [inline src/components/MenuDelivery/MenuDelivery.jsx]: Why don't we call this `ActionsButton` instead? MenuDelivery is very ambiguous 
```suggestion
const ActionsButton = () => {
```

File: tatuds (hunk 1 above)
Comment (pichar376) [reply tatuds]: ohh yes

File: src/components/MenuDelivery/MenuDelivery.jsx (hunk 3 above)
Comment (pichar376) [reply src/components/MenuDelivery/MenuDelivery.jsx]: wooow i didn't know this.

**Review by r3ivaj — APPROVED** [review]

**Pull request merged**